JSDemos/Demos/Lookup/Templates/description.md

```diff
@@ -1 +1,9 @@
-This demo shows how you can customize the input field and the items of&nbsp;the Lookup component.
\ No newline at end of file
+The Lookup contains properties to specify custom templates for the following elements:
```

Comment:
I'm not sure that 'contains properties' is a correct term. How about the following: 
The Lookup allows you to use the following properties to specify custom templates for the lookup elements:

Reply:
"You can use the following properties to specify custom templates for these Lookup elements:" - used this to avoid repeating "to".